electrum/commands.py

```diff
@@ -710,6 +710,34 @@ async def onchain_history(self, year=None, show_addresses=False, show_fiat=False
 
         return json_normalize(wallet.get_detailed_history(**kwargs))
 
+    @command('wp')
+    async def bumpfee(self, tx, new_fee_rate, coins=None, strategies=None, password=None, unsigned=False, wallet: Abstract_Wallet = None):
+        """ Bump the Fee for an unconfirmed Transaction """
+        tx = Transaction(tx)
+        domain_coins = coins.split(',') if coins else None
+        strategies = strategies.split(',') if strategies else None
+        bumpfee_strategies = None
+        if strategies is not None:
+            bumpfee_strategies = []
+            for strategy in strategies:
+                if strategy == 'CoinChooser':
+                    bumpfee_strategies.append(BumpFeeStrategy.COINCHOOSER)
+                elif strategy == 'DecreaseChange':
+                    bumpfee_strategies.append(BumpFeeStrategy.DECREASE_CHANGE)
+                elif strategy == 'DecreasePayment':
+                    bumpfee_strategies.append(BumpFeeStrategy.DECREASE_PAYMENT)
+                else:
+                    return "Invalid Choice of Strategies."
+        new_tx = wallet.bump_fee(
+            tx=tx,
+            txid=tx.txid(),
+            coins=domain_coins,
```

Comment:
I think `domain_coins` here has the wrong type. It is proably `List[str]` but `wallet.bump_fee` is expecting `Sequence[PartialTxInput]`.
Note how there is conversion from str in the payto case:
https://github.com/spesmilo/electrum/blob/a1f69fef97d820f7bcdfd6af5c7e7a1b1b704786/electrum/commands.py#L639
https://github.com/spesmilo/electrum/blob/a1f69fef97d820f7bcdfd6af5c7e7a1b1b704786/electrum/wallet.py#L2532-L2534